### ISSUE TITLE
Version and/or option to consent subjects to share undefaced data

### ISSUE THREAD

**Issue by effigies** [con/open-brain-consent#58]
An open repository of structural data with intact facial features could be a valuable resource. The most obvious use case is in developing defacing tools, permitting the validation of methods, quantifying the effects on downstream processing (see #53), or the automatic detection of defaced/undefaced images.

To obtain subject consent and ethics committee approval, I can see two paths:

1) A specific version of the OBC in which subjects consent to share undefaced data.
2) An addition to the standard OBC which allows subjects to opt-in to this additional release.

The former would allow studies to be planned entirely around releasing undefaced datasets. The latter would allow for an accumulation of some subjects over many studies.

One point to note in any disclosure is that it's more-or-less irrevocable. Since brains are like fingerprints, a defaced subject in one study can be associated with an undefaced subject in another study, and a subject would effectively be consenting to release all past and future contributions to open datasets.

How do people feel about this?

**Comment by PeerHerholz** [comment]
I think this is a very important point and I completely agree with the reasoning and your example use cases! I hope you don't mind if I bring the other folks of the `GDPR` play group into the loop as we discussed those things quite a bit. @CPernet, @jsheunis, @robertoostenveld

**Comment by jsheunis** [comment]
I agree, the value of this for methods development and validation would be great. Regarding GDPR I think this gets to one of the main points of contention, whether it's "allowed" to share the personal, identifiable data of a study participant if they have been clearly informed about the risks and benefits involved and have agreed to sharing. I think different DPOs will take different stances. Some might say the risk is too high, even if (some) participants might agree to sharing after being informed. In the case of shared undefaced images, the risk of identification would be higher - but by how much? Is it useful to try and quantify this more than just saying "higher"?

Regarding the suggested changes, I would say both could be useful.

**Comment by CPernet** [comment]
in principle this is feasable but I'd say not under a CC0 or CCBY licence - something like the DUA we have would work, offering extra protection

**Comment by robertoostenveld** [comment]
If I were on an IRB, I would not see why you need to share personal data (the brain anatomy) to study and improve defacing methods. Although, as a researcher, I would consider such a database as a "nice to have" and if participants and IRBs were to approve, then I don't see a  problem. 

Subsequently taking the perspective of a grant reviewer that looks at someones proposal to record a lot of new non-defaced anatomical MRIs.... There are actually already quite some non-defaced MRI data our there, since a lot of the earlier sharing efforts (including openfmri) did not deface. So rather than collecting new data for that purpose, I would recommend making a metadatabase that gathers information about the already available datasets. Or e.g. add it to openneuro/BIDS as metadata keyword. 

For the [BIG](https://www.ru.nl/donders/@1165458/big-database-understand-genetics-brain/) project at the Donders (running >10 years, now includes >4000 MRIs) we have in the past shared facial features (actually larynx, mouth, and nose) in a study that investigated speech and the vocal tract. For that, we used what we then referred to as a "Frankenstein" approach, where we replaced the brain of the individual participants' MRIs with that of the MNI canonical template. 

So rather than defacing, we used a "debraining" technique. And the brain was replaced with something that was similar to allow regular (SPM based) brain-based coregistration procedures still to work. Something like this (from an early try-out version that I found as an example in an old email)

![Mail Attachment](https://user-images.githubusercontent.com/899043/84009311-5e7ffa00-a973-11ea-8c1a-d11582edf28a.png)


For BIG (which basically relies on participants to our typical cognitive neuroimaging studies agreeing that their anatomical and rs-fMRI are also stored in the BIG database, along with collecting a cheek-swab sample for genetics) we do have an additional consent form. Furthermore, access to the BIG database (currently only for internal researchers) follows a review of a project proposal, i.e. it is not open for all, and only data relevant to the project proposal will be shared. The proposal for this study was approved, and I guess the study  is somewhere published by [the author](https://scholar.google.com/citations?user=JbQKSMoAAAAJ&hl=en).
